modules/nation_notes/nn_acl_upgrade.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+* Add GACL sections for Nation Notes.
+
+* @package   LibreEHR
+* @author    Art Eaton <art@suncoastconnection.com>
+* @link      http://LibreHealth.io
+*/
+
+$ignoreAuth = true; 
+
+require_once('../../interface/globals.php');
```

Comment:
I added the GL value and a variable into globals.inc.  I also cleaned up the ZEND modules stuff, and re-organized them to a location in the code that makes more sense (like where we are making path globals).
Next round will be me removing these relative paths and using the global variables.